pages/dao/[symbol]/proposal/[pk]/index.tsx

```diff
@@ -172,8 +175,13 @@ const Proposal = () => {
                   <VoteResultStatus />
                 </div>
               )}
-              <VoteResults proposal={proposal.account} />
-              {proposal && (
+
+              {!isMulti ?
```

Comment:
for the sake of consistency let's use `isMulti ?` not `!isMulti ?`, just gets confusing to swap booleans in your head